tests/Agent/UnitTests/Core.UnitTest/Segments/MethodSegmentDataTests.cs

```diff
@@ -17,7 +17,9 @@ public class MethodSegmentDataTests
     {
         #region IsCombinableWith
 
+#pragma warning disable NUnit1028 // The non-test method is public
```

Comment:
Should we submit a tech debt ticket for this pragma too?

Reply:
Refactored